DXMainClient/DXGUI/Multiplayer/GameListBox.cs

```diff
@@ -332,6 +359,19 @@ public override void Draw(GameTime gameTime)
                         height, txPasswordedGame.Width, txPasswordedGame.Height),
                         Color.White);
                 }
+                else
+                {
+                    Logger.Log("GameDifficulty **  Drawing game difficulty icon: " + hostedGame.GameDifficulty);
+                    Texture2D txGameDifficulty = txGameDifficultyIcons[hostedGame.GameDifficulty];
+
+                    if (txGameDifficulty != null)
+                    {
+                        DrawTexture(txGameDifficulty,
+                            new Rectangle(Width - txGameDifficulty.Width - TextBorderDistance - (scrollBarDrawn ? ScrollBar.Width : 0),
+                            height, txGameDifficulty.Width, txGameDifficulty.Height),
+                            Color.White);
```

Comment:
How does this look with locked/passworded/incompatible game icons?

Reply:
For passworded games it doesn't show the icon. No need. 
Incompatible/locked icons are on the left hand side.
The only icons on the right would be the difficulty icons OR password.

Reply:
Right, but in future we should generalize this like RA2/YR icons implementation I've told you about earlier. The one where a broadcasted game option could add an icon, and every game option including those ones could draw an icon not impacting other game options.